ome_zarr/writer.py

```diff
@@ -181,30 +181,38 @@ def write_multiscale(
     """
     Write a pyramid with multiscale metadata to disk.
 
-    pyramid: List of np.ndarray
-      the image data to save. Largest level first
-      All image arrays MUST be up to 5-dimensional with dimensions
-      ordered (t, c, z, y, x)
-    group: zarr.Group
-      the group within the zarr store to store the data in
-    chunks: int or tuple of ints,
-      Size of the saved chunks to store the image.
-      This argument is deprecated and will be removed in a future version.
-      Use storage_options instead.
-    fmt: Format
-      The format of the ome_zarr data which should be used.
-      Defaults to the most current.
-    axes: str or list of str or list of dict
-      List of axes dicts, or names. Not needed for v0.1 or v0.2
-      or if 2D. Otherwise this must be provided
-    coordinate_transformations: 2Dlist of dict
-      For each path, we have a List of transformation Dicts.
-      Each list of dicts are added to each datasets in order
-      and must include a 'scale' transform.
-    storage_options: dict or list of dict
-      Options to be passed on to the storage backend. A list would need to match
-      the number of datasets in a multiresolution pyramid. One can provide
-      different chunk size for each level of a pyramind using this option.
+    :type pyramid: list of :class:`np.ndarray`
```

Comment:
The `np.ndarray` class here and elsewhere don't create links to the `numpy` docs, whereas `numpy.ndarray` probably would?